### ISSUE TITLE
Bug: Labels for CheckBox goes outside the view

### ISSUE THREAD

**Issue by matsovevada** [enonic/app-contentstudio#5284]
If i have a longer label on a checkbox it disappears after the view. It looks like the view opens up to fit the longer text, but if i drag the view to the left to get more space for the preview some of the text disappears.

**Comment by alansemenov** [comment]
@matsovevada can you provide a screenshot? we have lots of different views where we are using checkboxes, so it's a bit difficult do pinpoint which of them you are referring to.

**Comment by matsovevada** [comment]
Yes, of course! And this is even worse on screens with lower resolution, fyi :)

![image](https://user-images.githubusercontent.com/55790581/195520565-1dd086ff-1b61-462a-b005-1fdd6ab908c6.png)
